onboarding_auth/onboardingAuthDomain/src/main/java/com/alexafit/onboardingauthdomain/model/domain/mapper/ValidateEmailResult.kt

```diff
@@ -0,0 +1,6 @@
+package com.alexafit.onboardingauthdomain.model.domain.mapper
+
+data class ValidateEmailResult(
+    val emailResult: String = "",
```

Comment:
What exactly would `emailResult` mean here? From your previous PR I remember it's the trimmed email, but I don't think you'd understand that just from reading this name here. I also don't find a perfect name, but maybe something like `adjustedEmail` or so works a bit better? If you disagree, feel free to keep it